com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/MeshsetBuilder.java

```diff
@@ -95,7 +104,68 @@ public void build(Task<Void> task) throws CompileExceptionError, IOException {
         animationSetBuilder.build().writeTo(out);
         out.close();
         task.output(2).setContent(out.toByteArray());
+    }
+
+    @Override
+    public void build(Task<Void> task) throws CompileExceptionError, IOException {
+
+        String suffix = BuilderUtil.getSuffix(task.input(0).getPath());
+
+        if (suffix.equals("dae")) {
+            buildCollada(task); // Until our model importer supports collada
+            return;
+        }
+
+        ModelImporter.Options options = new ModelImporter.Options();
+        ModelImporter.Scene scene = ModelUtil.loadScene(task.input(0).getContent(), task.input(0).getPath(), options);
```

Comment:
We currently don't have any options, but it should be easy to add if we need to.